tests/testsuite/fix.rs

```diff
@@ -887,13 +901,7 @@ fn prepare_for_already_on_latest_stable() {
         .build();
 
     p.cargo("fix --edition --allow-no-vcs")
-        .with_stderr_contains("[CHECKING] foo [..]")
-        .with_stderr_contains(&format!(
-            "\
-[WARNING] `src/lib.rs` is already on the latest edition ({latest_stable}), unable to migrate further
-",
-            latest_stable = latest_stable
-        ))
+        .with_stderr_data(str![])
         .run();
```

Comment:
I'm not sure why this isn't being overwritten. Has this functionality been tested at all?

Reply:
No. I think we can add deprecated and leave off for now.

https://github.com/rust-lang/cargo/blob/c7cb45c0fdb65fc1d0252ecf4d7a8672667bc5a5/tests/testsuite/fix.rs#L881-L885